packaging/debian/postinst

```diff
@@ -42,37 +51,24 @@ case "$1" in
 	external_addr="$RET"
 	db_get storagenode/storagedir
 	storagedir="$RET"
-	db_get storagenode/old_storagedir
-	old_storagedir="$RET"
 	db_get storagenode/diskspace
 	diskspace="$RET"
-	db_get storagenode/bandwidth
-	bandwidth="$RET"
-
-	function get_binary {
-	    binary=$1
-	    tmpfile=$(mktemp /tmp/${binary}.zip.XXXXXX)
-	    wget -O $tmpfile ${BINARIES_SERVER}/${binary}_linux_amd64.zip
-	    unzip $tmpfile -d /var/lib/storagenode
-	    mv /var/lib/storagenode/${binary}_linux_amd64 /var/lib/storagenode/${binary}
-	    rm $tmpfile
-	}
 
 	for binary in "storagenode" "storagenode-updater"; do
-	    if [[ ! -f "/var/lib/storagenode/${binary}" ]]; then
+	    if [ ! -f "/var/lib/storagenode/${binary}" ]; then
 	        echo "Downloading ${binary} binary..."
 	        get_binary $binary
 	    fi
 	done
 
-	if [[ "$old_storagedir" != "" ]] && [[ "$storagedir" != "$old_storagedir" ]]; then
+	if [ ! -z "$old_storagedir" ] && [ "$storagedir" != "$old_storagedir" ]]; then
```

Comment:
@Fadila82 could you pls explain what is the purpose of this block of code? Seems like `old_storagedir` is not set and not used anywhere except for this block which just print some info

Reply:
If I remember well, I had put this to not forget that in case of reinstallation, we had to check that we're using the same storage directory, and if not (which is not done now), move things from the old to the new one. 
But right now, I'm not sure it's the best thing to do.

Reply:
Make sure to take off the extra `]` near the end of the line if we decide to keep this.